client/navigation/components/header/index.js

```diff
@@ -19,17 +19,35 @@ const Header = () => {
 	const siteTitle = getSetting( 'siteTitle', '' );
 	const siteUrl = getSetting( 'siteUrl', '' );
 	const isScrolled = useIsScrolled();
+	const navClasses = {
+		folded: 'is-folded',
```

Comment:
Existed prior to this PR, but realizing that these classes are generic and do not describe the `body` but instead describe the nav.  Maybe we should update these to `is-wc-nav-folded` and `is-wc-nav-expanded` to more accurately describe what's happening and avoid conflicts.